Core/Inc/dashboard.h

```diff
@@ -1,22 +1,25 @@
 /*INCLUDES*/
 
+#include "bsp.h"
+#include "ili9488.h"
```

Comment:
In future improvements of the code you should end up in a state where the file that contains the control code doesn't depend on level drivers, i.e. you will lose the `#include "ili9488.h` dependency in `dashboard.c` because all the code you will need is in either LVGL stuff or `bsp.h`.

You can close this comment, just keep in mind it.